src/main/resources/openapiv2.json

```diff
@@ -0,0 +1,876 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Strimzi HTTP Bridge for Apache Kafka",
+        "description": "The Strimzi HTTP Bridge for Apache Kafka provides a REST API for integrating client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol. The HTTP Bridge can be deployed on bare metal or in a container.",
```

Comment:
"... interacting HTTP based client applications ..."
" ... or in a container running on Kubernetes or OpenShift platform"

Reply:
interacting with HTTP-based client applications